aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -1103,6 +1103,150 @@ Tensor clone(const Tensor& src, c10::optional<c10::MemoryFormat> optional_memory
   return self;
 }
 
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ covariations ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tensor cov(const Tensor& self, const Tensor& other, const Tensor& fweights, 
```

Comment:
Eventually we should find a better home for cov than TensorFactories.cpp, which is intended for operations like torch.empty.